packages/preferences/src/browser/views/components/single-preference-wrapper.tsx

```diff
@@ -139,22 +139,21 @@ export class SinglePreferenceWrapper extends React.Component<SinglePreferenceWra
         service: PreferenceService): React.ReactNode[] | undefined {
         if (preferenceValuesInAllScopes) {
             return ['User', 'Workspace'].map((scope: 'User' | 'Workspace') => {
-                const matchingScope = PreferenceScope[scope];
-                if (currentScope !== matchingScope) {
+                const otherScope = PreferenceScope[scope];
+                if (currentScope !== otherScope) {
                     const info = service.inspect<PreferenceItem>(id);
                     if (!info) {
                         return;
                     }
 
                     const defaultValue = info.defaultValue;
                     const currentValue = this.getValueByScope(info, currentScope);
-                    const matchingValue = this.getValueByScope(info, matchingScope);
-
-                    if (matchingValue !== undefined) {
+                    const otherValue = this.getValueByScope(info, otherScope);
+                    if (otherValue !== undefined && otherValue !== defaultValue) {
```

Comment:
@vince-fugnitto, I changed this line to check that it `!== defaultValue` because I was getting a bug where if I changed the value in `User` and not in `Workspace`, I was still getting a message in `User` saying that it was modified in `Workspace`.

Reply:
I think the issue there was that it's possible to explicitly set the value of the preference to be the same as the default, in which case the `!== undefined` check would pass, but it still wouldn't really have been modified away from the default.